.github/workflows/build-test-and-publish.yml

```diff
@@ -166,27 +166,29 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     name: Dockerize and publish
-    needs: [ version, build, test ]
+    needs: [version, build, test]
+
+    env:
+      AWS_REGION: eu-central-1
+      IMAGE_REPO: backoffice
 
     steps:
       - name: Fetch version artifact
         uses: actions/download-artifact@v2
         with:
           name: version
-      
+
       - name: Fetch build artifact
         uses: actions/download-artifact@v2
         with:
           name: backoffice
-      
+
       - name: Untar application files
         run: tar -xvf app.tar .
 
       - name: Set env
         run: |
-          echo ::set-env name=VERSION::$(cat ./version.txt)
-          echo ::set-env name=IMAGE_REPO::backoffice
-          echo ::set-env name=AWS_REGION::eu-central-1
+          echo "VERSION=$(cat ./version.txt)" >> ${GITHUB_ENV}
```

Comment:
This is the new way to set environment variables that evaluate shell expressions.